### PR TITLE
Migrate to assembly 1.2.0

### PR DESCRIPTION
**Pull request by samanpwbb**
This PR introduces the minimum necessary changes in order to support assembly 1.1.0 while maintaining roughly the same visual styles. Some form elements have changed slightly, and many colors have changed slightly.

This is ready for review now that mbx-assembly 1.0.0 has been released

### CODE REVIEW

**Comment by samanpwbb** [comment]
Draft until we've deployed https://github.com/mapbox/mbx-assembly/pull/54

File: src/components/control-alert/examples/control-alert-example-options.js [hunk 1]
```diff
@@ -27,10 +27,10 @@ export default class Example extends React.Component {
     if (showAlert) return null;
 
     return (
-      <div className="flex-parent flex-parent--end-main mr18 mb18">
+      <div className="flex flex--end-main mr18 mb18">
         <button
           aria-label="Show alert again"
-          className="color-gray-dark color-blue-on-hover flex-child"
+          className="color-gray-dark color-blue-on-hover "
```
Comment (tristen) [inline src/components/control-alert/examples/control-alert-example-options.js:33]:
```suggestion
          className="color-gray-dark color-blue-on-hover"
```

File: src/components/control-card/examples/control-card-example-options.js [hunk 2]
```diff
@@ -27,10 +27,10 @@ export default class Example extends React.Component {
     if (showCard) return null;
 
     return (
-      <div className="flex-parent flex-parent--end-main mr18 mb18">
+      <div className="flex flex--end-main mr18 mb18">
         <button
           aria-label="Show card"
-          className="color-gray-dark color-blue-on-hover flex-child"
+          className="color-gray-dark color-blue-on-hover "
```
Comment (tristen) [inline src/components/control-card/examples/control-card-example-options.js:33]:
```suggestion
          className="color-gray-dark color-blue-on-hover"
```

File: src/components/control-switch/__tests__/control-switch-test-cases.js [hunk 3]
```diff
@@ -47,7 +47,7 @@ testCases.allOptions = {
     optional: true,
     label: <span>All options</span>,
     themeControlSwitchContainer: 'txt-s',
-    themeControlSwitch: 'switch--red switch--dot-red',
+    themeControlSwitch: 'switch--red switch--dot-yellow',
```
Comment (tristen) [inline src/components/control-switch/__tests__/control-switch-test-cases.js:50]: 🎪

File: src/components/underline-tabs/underline-tabs.js [hunk 4]
```diff
@@ -17,15 +17,15 @@ class UnderlineTabs extends React.Component {
...
+    const containerClasses = classnames('flex txt-nowrap unselectable', {
       'txt-bold': props.bold,
       'txt-s': medium,
       'txt-xs': small
     });
 
     const itemEls = props.items.map((item, index) => {
       const first = index === 0;
-      const layoutClasses = classnames('flex-child', {
+      const layoutClasses = classnames('', {
```
Comment (tristen) [inline src/components/underline-tabs/underline-tabs.js:28]:
```suggestion
      const layoutClasses = classnames({
```

File: src/docs/components/sidebar.js [hunk 5]
```diff
@@ -20,15 +20,15 @@ export default class Sidebar extends React.Component {
...
           href="https://github.com/mapbox/mr-ui"
           target="_blank"
           rel="noopener noreferrer"
         >
-          <span className="flex-child mr6">
+          <span className="mr6">
             <Icon name="github" />
           </span>
-          <span className="flex-child">View on GitHub</span>
+          <span className="">View on GitHub</span>
```
Comment (tristen) [inline src/docs/components/sidebar.js:31]:
```suggestion
         View on GitHub
```

**Review by tristen — CHANGES_REQUESTED** [review]

File: src/components/control-checkbox-set/control-checkbox-set.js [hunk 6]
```diff
@@ -63,9 +63,8 @@ export default class ControlCheckboxSet extends React.Component {
     optional: false,
     autoFocus: false,
     icon: 'check',
-    themeCheckboxContainer: 'txt-s block mb6 flex-parent',
-    themeCheckbox:
-      'mr6 inline-block checkbox--blue checkbox--s-label flex-child'
+    themeCheckboxContainer: 'txt-s block mb6 flex',
+    themeCheckbox: 'mr6 inline-block checkbox--blue checkbox--s-label '
```
Comment (katydecorah) [inline src/components/control-checkbox-set/control-checkbox-set.js:67]:
```suggestion
    themeCheckbox: 'mr6 inline-block checkbox--blue checkbox--s-label'
```

**Review by katydecorah — COMMENTED** [review]
Looking good so far! I found a few small-ish suggestions.

**Comment by samanpwbb** [comment]
@tristen / @katydecorah this is ready for another look. Note that I added some commits that introduce some new sections to the docs site - would be great to get some 👁️ on that work too!

Screen Shot 2021-05-25 at 1 25 45 PM

**Review by tristen — APPROVED** [review]
Changes look good to me! Approving to unblock but the layout for the main content is condensing the name column in some places:

Screen Shot 2021-05-26 at 12 10 49 PM

**Pull request merged**